workflows/mark02.smk

```diff
@@ -188,6 +188,11 @@ def make_all_mark02():
                             init_trees_ids[1]) + "_i" + str(init_trees_ids[2]) + "_i" + str(init_trees_ids[3]) + "/rhat.svg"
                     )
 
+                    # also effective sample size
+                    filepaths.append(
```

Comment:
In the future projects I'd suggest to create an auxiliary function which takes these arguments and returns a formatted string (or `pathlib.Path`).

Reply:
Good idea. Thanks.